Projects/UOContent/Mobiles/Animals/Mounts/SwampDragon.cs

```diff
@@ -49,56 +45,59 @@ public SwampDragon() : base(0x31A, 0x3EBD, AIType.AI_Melee, FightMode.Aggressor)
             MinTameSkill = 93.9;
         }
 
-        public SwampDragon(Serial serial) : base(serial)
-        {
-        }
-
         public override string CorpseName => "a swamp dragon corpse";
 
         [CommandProperty(AccessLevel.GameMaster)]
+        [SerializableProperty(1)]
```

Comment:
This can be made into a field:

```cs
[InvalidateProperties]
[SerializableField(1)]
private string _bardingCraftedBy;
```

Same with barding exceptional.

Reply:
do you want to keep [CommandProperty(AccessLevel.GameMaster)]?